dgbpy/bokeh_machine_learning/main.py

```diff
@@ -146,22 +147,22 @@ def sendmsg(self, msgnm):
 
     trainpanel = Panel(title=traintabnm)
     parameterspanel = Panel(title=paramtabnm)
-    mainpanel = Tabs(tabs=[trainpanel,parameterspanel])
+    adparameterspanel = Panel(title=adparamtabnm, disabled = True)
```

Comment:
There should be no need to disable it, this is done by setting the active platform on line 408